helmctlr/helm_test.go

```diff
@@ -134,9 +152,12 @@ func TestResourceAddedHappyPath(t *testing.T) {
 		create:   1,
 		releases: 1,
 	}
+
+	currTimestamp := getTimestamp()
 	assertCounters(t, ct, func() {
 		testController.ResourceAdded(testResource)
 	})
+	assertTimestamp(t, "releases_last_create_timestamp_utc_seconds", greaterThan, currTimestamp)
```

Comment:
Can we combine this with the `assertCounters` wrapper. Maybe make it `assertMetrics` instead. I'm not a fan of having to get current time and then this is also only validating the one we expect to change changes, it doesn't do anything to ensure the others remained the same.